runbotics/runbotics-scheduler/src/scheduler-database/schedule-process/dto/create-schedule-process.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const createScheduleProcessSchema = z.object({
+    cron: z.string(),
```

Comment:
It is a good idea, however I think I won't add it here for now - because as I was testing it, I wasn't really sure for regex and I need more time to think about it.